ssl/statem/statem_srvr.c

```diff
@@ -1956,9 +1987,9 @@ WORK_STATE tls_post_process_client_hello(SSL *s, WORK_STATE wst)
         wst = WORK_MORE_B;
     }
     if (wst == WORK_MORE_B) {
-        if (!s->hit || s->hello_retry_request) {
+        if (!s->hit || SSL_IS_TLS13(s)) {
```

Comment:
Likewise for this hello_retry_request=>SSL_IS_TLS13() conversion; is it unrelated cleanup?